Bad formatting in pidfile utility

This test fails on master (at least locally on my computer):

```
# github.com/cilium/cilium/pkg/pidfile
pkg/pidfile/pidfile.go:76: Errorf format %s has arg pid of wrong type int
make: *** [Makefile:92: unit-tests] Error 1
```

Which is understandable, because this formatting is indeed wrong (we pass 2 arguments with `int` and `error`/`string` types, the format string expects only one `string`):
https://github.com/cilium/cilium/blob/master/pkg/pidfile/pidfile.go#L76

--- a/pkg/pidfile/pidfile.go
+++ b/pkg/pidfile/pidfile.go
@@ -73,7 +73,7 @@ func kill(buf []byte) error {
 		log.WithError(err).Debug("Ignoring process kill failure")
 	}
 	if err := oldProc.Release(); err != nil {
-		return fmt.Errorf("Couldn't release process: %s", pid, err)
+		return fmt.Errorf("Couldn't release process %d: %s", pid, err)
 	}
 	return nil
 }

